clarity/src/vm/docs/mod.rs

```diff
@@ -2726,7 +2723,7 @@ fn make_all_api_reference() -> ReferenceAPIs {
         .filter_map(make_keyword_reference)
         .collect();
 
-    keywords.sort_by(|x, y| x.name.cmp(&y.name));
+    keywords.sort_by(|x, y| x.name.cmp(y.name));
```

Comment:
This could be simplified further:

```suggestion
    keywords.sort_by_key(|x| x.name);
```